jekyll/_cci2/triggers-overview.adoc

```diff
@@ -52,9 +52,6 @@ Once your Bitbucket Cloud project is set up in CircleCI, a pipeline is triggered
 --
 
 [tab.event-trigger.BitBucket_Data_Center]
---
-include::../_includes/partials/notes/bitbucket-data-center-closed-preview.adoc[]
```

Comment:
All good apart from this one would end up being an empty tab. So I guess we need to add something about BBDC triggers. Is there anything unique to point out?

Reply:
done!